RecoLocalTracker/SiStripClusterizer/interface/StripClusterizerAlgorithm.h

```diff
@@ -27,7 +27,7 @@ class StripClusterizerAlgorithm {
     float noise(const uint16_t& strip) const { return SiStripNoises::getNoise( strip, noiseRange ); }
     float gain(const uint16_t& strip)  const { return SiStripGain::getStripGain( strip, gainRange ); }
     bool bad(const uint16_t& strip)    const { return quality->IsStripBad( qualityRange, strip ); }
-    bool allBadBetween(uint16_t L, const uint16_t& R) const { while( ++L < R  &&  bad(L) ); return L == R; }
+    bool allBadBetween(uint16_t L, const uint16_t& R) const { while( ++L < R  &&  bad(L)) {}; { return L == R;} }
```

Comment:
The braces around the `return` do not seem to be needed.

Reply:
Sorry. Left over from when I was trying to figure out what they were doing.